src/pages/Login/Login.js

```diff
@@ -1,29 +1,45 @@
 import React, { useState } from 'react';
 import './Login.scss';
 import { useNavigate } from 'react-router-dom';
-
 const Login = () => {
   const navigate = useNavigate();
   const [inputValue, setInputValue] = useState({
-    id: '',
-    pw: '',
+    email: '',
+    password: '',
   });
-  const { id, pw } = inputValue;
-
+  const { email, password } = inputValue;
   const handleInput = e => {
     const { name, value } = e.target;
-    setInputValue(prev => ({ ...prev, [name]: value }));
+    setInputValue({ ...inputValue, [name]: value });
   };
-  const isValid = id.includes('@' && '.com') && pw.length >= 5;
-
+  const isValid = email.includes('@' && '.com') && password.length >= 5;
   const goToMain = () => {
+    console.log(email, password);
     if (isValid) {
+      fetch('http://10.58.52.209:3000/users/signin', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json;charset=utf-8', //필수로 넣어야함
+        },
+        body: JSON.stringify({
+          // 자바스크립트를 json 형태로 보내는 것
+          email: inputValue.email,
+          password: inputValue.password,
+        }),
```

Comment:
```suggestion
        body: JSON.stringify(inputValue),
```
결국 해당 값은 inputValue에 담겨진 값이기 때문에
이렇게 넣을수도 있겠습니다!